sci-libs/openblas/openblas-0.3.6.ebuild

```diff
@@ -66,6 +72,11 @@ src_install () {
 		doins interface/liblapack.so.3
 		dosym liblapack.so.3 usr/$(get_libdir)/lapack/openblas/liblapack.so
 	fi
+
+	if use index-64bit; then
+		insinto /usr/$(get_libdir)/
+		doins "${S}-index-64bit"/libopenblas64*
```

Comment:
1. Should the library just be installed under /usr/$(get_libdir)/ , instead of a subdirectory `libopenblas64`?

1. Please use `dolib.so` helpers.

Reply:
These libraries can be installed to the public directory.

Reply:
I mean, here we use `dolib.so` instead of `doins`.  https://devmanual.gentoo.org/function-reference/install-functions/

Reply:
> I mean, here we use `dolib.so` instead of `doins`. https://devmanual.gentoo.org/function-reference/install-functions/

This was already fixed, see the diff: https://github.com/gentoo/gentoo/compare/e5ac866f20a88696742bcf2dcc86ae0a2a6c3748..97f4afd09cf88dcd99eb4862113d2e6247e0f74c